app/src/components/domain/PerAssessmentSummary/i18n.json

```diff
@@ -1,7 +1,7 @@
 {
-    "namespace": "common",
+    "namespace": "perAssessmentSummary",
```

Comment:
@frozenhelium What is the guideline for the namespace for components inside domain?

Reply:
We're using component name for now

Reply:
Reverting to "common" for the time being.